app/src/main/java/ch/epfl/sweng/studyup/utils/adapters/QuestListViewAdapterStudent.java

```diff
@@ -0,0 +1,54 @@
+package ch.epfl.sweng.studyup.utils.adapters;
```

Comment:
This is an old file, should be deleted

Reply:
This is a package containing the list view for Questions and Special Quests.